scripts/actions/send-and-update-translation-queue.js

```diff
@@ -190,8 +206,14 @@ const sendContentToVendor = async (content, accessToken) => {
   });
 
   const batchResponses = await Promise.all(batchRequests);
+  const initialStatus = 'PENDING';
+  for (const response of batchResponses) {
+    addJob(response.job_uid, response.batch_uid, initialStatus);
```

Comment:
this will also need await. additionally, the status we pass in should be integer value, not text.

Reply:
yessir